test/testdrive/mzcompose.yml

```diff
@@ -8,6 +8,58 @@
 # by the Apache License, Version 2.0.
 
 version: '3.7'
+
+mzworkflows:
+  # Run tests, requires aws credentials in the environment
+  ci:
+    env:
+      AWS_REGION: us-east-2
+      AWS_ENDPOINT: ''
+      # The first TD_TEST parameter is automatically appended to
+      # test/testdrive, but the second is just a plain string
+      TD_TEST: ${TD_TEST:-*.td test/testdrive/esoteric/*.td}
```

Comment:
This should be changed, I think either by making testdrive itself handle `TD_TEST` or by adding a `command_target` key to mzcompose's `workflow`

Reply:
I definitely don't think `testdrive` the binary should be looking at an environment variable that exists just to work around limitations in mzcompose! I'm not sure I follow the `command_target` proposal either.

Can we instead just change the testdrive entrypoint to `cd` into `test/testdrive` first, and drop the `test/testdrive` prefix in the default command? Then everything will be automatically relative to `test/testdrive`.

Reply:
I'll try that, yeah.

Reply:
> I'm not sure I follow the command_target proposal either.

The idea is we could write:

```yaml
mzworkflows:
  local:
    command_target: testdrive
    steps:
      - step: run
        service: testdrive
```

and then if you run `mzcompose run ci --ci-output joins.td` the `--ci-output joins.td` will get
forwarded to `testdrive`. This is forward-compatible with a hypothetical `--command=testdrive='--ci-output joints.td'` thing that we could also do.

Reply:
`TD_TEST` with a directory change works, though, so I'll leave that hypothetical hunk of work off for a future person to think about.